config/v1/0000_10_config-operator_01_build.crd.yaml

```diff
@@ -216,6 +216,11 @@ spec:
                                 description: Specify whether the Secret or its key
                                   must be defined
                                 type: boolean
+                forcePull:
+                  description: ForcePull denotes whether or not images (false) should
+                    be looked for on the local node or (true) should be refreshed
+                    from the registry to which the image stream points
```

Comment:
technically they are always looked for/used from the node (well, ignoring the fact that today builds *never* look at/use node content).  forcepull determines whether or not they first check the external registry to see what the latest SHA is for the image tag, vs just using whatever SHA happens to exist for that tag locally.  From there it will use the local content if the SHAs are present and pull if not, again regardless of whether force pull is true or not.

also maybe it's personal preference, but i'd put the `true` and `false` after the description, not before.  i had trouble parsing the sentences as they are because i didn't know what `false` referred to (because i hadn't read it yet).

Reply:
@bparees Thanks.  Updated in current push.